.github/workflows/publish-docs-site.yml

```diff
@@ -11,7 +11,7 @@ jobs:
         name: Build & publish site
         runs-on: ubuntu-latest
         # Run the job if manually triggered or if the commit message includes '#publish' & the check suite has passed or if the commit message includes 'docs' and the check suite has passed
-        if: github.event_name == 'workflow_dispatch' || contains(github.event.head_commit.message, '#publish') || contains(github.event.head_commit.message, 'docs')
+        if: github.event_name == 'workflow_dispatch' || contains(github.event.head_commit.message, '#publish') || contains(github.event.head_commit.message, 'docs:') || contains(github.event.head_commit.message, 'docs(')
```

Comment:
more explicit to avoid a commit message that has the word docs in the message. this ensures it is of the commit type docs